config/go-app-ussd_higherhealth_healthcheck.afr_ZA.json

```diff
@@ -0,0 +1,369 @@
+"Please select your preferred language.\n"
+"Reply\n"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:188
+msgid "Afrikaans"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:189
+msgid "English"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:190
+msgid "Sotho"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:191
+msgid "Xhosa"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:192
+msgid "Zulu"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:203
+msgid ""
+"The HIGHER HEALTH HealthCheck is your risk assessment tool. Help us by "
+"answering a few questions about you and your health.\n"
+"\n"
+"Reply"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:209
+msgid "This service works best when you select numbers from the list"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:212
+msgid "START"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:219
+msgid ""
+"Confirm that you're responsible for your medical care & treatment. This "
+"service only provides info.\n"
+"\n"
+"Reply"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:225
+msgid ""
+"Please use numbers from list. Confirm that u're responsible for ur medical "
+"care & treatment. This service only provides info.\n"
+"\n"
+"Reply"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:233
+#: go-app-ussd_higherhealth_healthcheck.js:372
+#: go-app-ussd_higherhealth_healthcheck.js:394
+#: go-app-ussd_higherhealth_healthcheck.js:416
+#: go-app-ussd_higherhealth_healthcheck.js:438
+#: go-app-ussd_higherhealth_healthcheck.js:460
+#: go-app-ussd_higherhealth_healthcheck.js:485
+msgid "YES"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:234
+#: go-app-ussd_higherhealth_healthcheck.js:373
+#: go-app-ussd_higherhealth_healthcheck.js:395
+#: go-app-ussd_higherhealth_healthcheck.js:417
+#: go-app-ussd_higherhealth_healthcheck.js:439
+#: go-app-ussd_higherhealth_healthcheck.js:461
+#: go-app-ussd_higherhealth_healthcheck.js:486
+msgid "NO"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:235
+msgid "MORE INFO"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:242
+msgid ""
+"You can return to this service at any time. Remember, if you think you have "
+"COVID-19 STAY HOME, avoid contact with other people and self-isolate."
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:252
+msgid ""
+"It's not a substitute for professional medical advice/diagnosis/treatment. "
+"Get a qualified health provider's advice about your medical condition/care."
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:257
+#: go-app-ussd_higherhealth_healthcheck.js:268
+msgid "Next"
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:263
+msgid ""
+"You confirm that you shouldn't disregard/delay seeking medical advice about "
+"treatment/care because of this service. Rely on info at your own risk."
+msgstr ""
+
+#: go-app-ussd_higherhealth_healthcheck.js:273
+msgid "Please TYPE your first name"
+msgstr ""
```

Comment:
I see we don't have any translations. I'm happy to add empty translations now, and add the translations at a later date.

But could you please also include the PO files used to generate these json files?